src/python/sortingview/gui/extensions/mountainview/MVSortingComparisonView/MVSortingComparisonView.tsx

```diff
@@ -0,0 +1,239 @@
+        openViewsDispatch({
+            type: 'SetViewArea',
+            viewId: view.viewId,
+            area
+        })
+    }, [openViewsDispatch])
+    const handleLaunchSortingComparisonView = useCallback((plugin: SortingComparisonViewPlugin) => {
+        openViewsDispatch({
+            type: 'AddView',
+            plugin,
+            label: plugin.label,
+            extraProps: {},
+            area: ''
+        })
+    }, [openViewsDispatch])
+    const width = props.width || 600
+    const height = props.height || 900
+    // const preprocessingIcon = <span style={{color: 'gray'}}><GraphicEq /></span>
+    const visibleUnitsIcon = <span style={{color: 'gray'}}><Visibility /></span>
+    const visibleElectrodesIcon = <span style={{color: 'gray'}}><GrainIcon /></span>
+    const launchIcon = <span style={{color: 'gray'}}><OpenInBrowserIcon /></span>
+    const curationIcon = <span style={{color: 'gray'}}><FontAwesomeIcon icon={faPencilAlt} /></span>
+    const optionsIcon = <span style={{color: 'gray'}}><Settings /></span>
+    const metricsIcon = <span style={{color: 'gray'}}><SquareFoot /></span>
+
+    const {workspaceUri} = useRoute()
+
+    const sortingComparisonViewProps = {...props}
+    return (
+        
+        <div className="MVSortingComparisonView MVSortingView">
+            <Splitter
+                width={width}
+                height={height}
+                initialPosition={initialLeftPanelWidth}
+            >
+                <div>
+                    {/* Launch */}
+                    <Expandable icon={launchIcon} label="Open views A" defaultExpanded={true} unmountOnExit={false}>
+                        <ViewLauncher
+                            onLaunchSortingView={handleLaunchSortingViewA}
+                            onLaunchRecordingView={handleLaunchRecordingView}
+                            onLaunchSortingUnitView={handleLaunchSortingUnitViewA}
+                            selection={props.selection1}
+                            hasSorting={true}
+                            sortingSelector="A"
+                        />
+                    </Expandable>
+                    <Expandable icon={launchIcon} label="Open views B" defaultExpanded={true} unmountOnExit={false}>
+                        <ViewLauncher
+                            onLaunchSortingView={handleLaunchSortingViewB}
+                            onLaunchRecordingView={handleLaunchRecordingView}
+                            onLaunchSortingUnitView={handleLaunchSortingUnitViewB}
+                            selection={props.selection2}
+                            hasSorting={true}
+                            sortingSelector="B"
+                        />
+                    </Expandable>
+                    <Expandable icon={launchIcon} label="Open comparison views" defaultExpanded={true} unmountOnExit={false}>
+                        <ComparisonViewLauncher
```

Comment:
This is the view launcher for the comparison views - which will be coming later. For example, widgets for comparing waveforms for best matching units between sorting A and sorting B.